packages/xo-server-transport-nagios/src/index.js

```diff
@@ -99,6 +121,14 @@ class XoServerNagios {
   configure(configuration) {
     this._conf = configuration
     this._key = Buffer.from(configuration.key, ENCODING)
+
+    const templateRules = {
+      '{vm.name_label}': vmNameLabel => vmNameLabel,
+      '{job.name}': (vmNameLabel, jobName) => jobName,
+    }
+
+    this._getHost = compileTemplate(this._conf.host, templateRules)
+    this._getService = compileTemplate(this._conf.service, templateRules)
```

Comment:
Don't save `host` and `service` in `this._conf`, use spread syntax to extract them, e.g.:

```js
configure({ host, service, ...configuration }) {
  this._conf = configuration
  ```